### ISSUE TITLE
SupportsShowChromedriverLogOption contains apparent spelling mistakes in method naming

### ISSUE THREAD

**Issue by fishey2** [appium/java-client#1967]
## Description

SupportsShowChromedriverLogOptions.java 

In my opinion, it seems that `doesDhowChromedriverLog()` and `setDhowChromedriverLog(boolean)` are incorrectly names and should instead be `doesShowChromedriverLog()` and `setShowChromedriverLog(boolean)`.

## Environment

* Java client build version or git revision if you use some snapshot: io.appium:java-client:8.5.1
* Appium server version or git revision if you use some snapshot: N/A
* Desktop OS/version used to run Appium if necessary: N/A
* Node.js version (unless using Appium.app|exe) or Appium CLI or Appium.app|exe: N/A
* Mobile platform/version under test: N/A
* Real device or emulator/simulator: N/A

## Details

```java
// SupportsShowChromedriverLogOptions.java


public interface SupportsShowChromedriverLogOption<T extends BaseOptions<T>>
        extends Capabilities, CanSetCapability<T> {
    String SHOW_CHROMEDRIVER_LOG_OPTION = "showChromedriverLog";

    /**
     * Enforces all the output from chromedriver binary to be
     * forwarded to the Appium server log.
     *
     * @return self instance for chaining.
     */
    default T showChromedriverLog() {
        return amend(SHOW_CHROMEDRIVER_LOG_OPTION, true);
    }

    /**
     * If set to true then all the output from chromedriver binary will be
     * forwarded to the Appium server log. false by default.
     *
     * @param value Whether to forward chromedriver output to the Appium server log.
     * @return self instance for chaining.
     */
    default T setDhowChromedriverLog(boolean value) {
        return amend(SHOW_CHROMEDRIVER_LOG_OPTION, value);
    }

    /**
     * Get whether to forward chromedriver output to the Appium server log.
     *
     * @return True or false.
     */
    default Optional<Boolean> doesDhowChromedriverLog() {
        return Optional.ofNullable(
                toSafeBoolean(getCapability(SHOW_CHROMEDRIVER_LOG_OPTION))
        );
    }
}
```

## Code To Reproduce Issue [ Good To Have ]

Not Applicable

## Exception Stacktraces

Not Applicable

## Link To Appium Logs

Not Applicable

**Comment by fishey2** [comment]
In addition, in `SupportsSimulatorTracePointerOption.java`, there is `doesSimulatorTracePointerd()`, whereas I think this should be `doesSimulatorTracePointer()` to bring inline with other interface definitions.

**Comment by fishey2** [comment]
In `SupportsCustomSslCertOption.java`, the getter is named `setCustomSSLCert()`, whereas I believe this should be `getCustomSSLCert()`.

**Comment by mykola-mokhnach** [comment]
Thanks @fishey2 
Yep, there are several typos there and it for sure it makes sense to fix them. We would be happy if you could propose a PR with a fix

**Comment by fishey2** [comment]
I have proposed a fix in PR - #1970, thanks @mykola-mokhnach
